applications/blackhole/src/blackhole_socket_callback.erl

```diff
@@ -74,7 +74,13 @@ close(SessionPid, SessionId, Context) ->
 %% @end
 %%--------------------------------------------------------------------
 -spec handle_info(ne_binary(), ne_binary(), any(), bh_context:context()) -> {'ok', bh_context:context()}.
+handle_info(_SessionPid, _SessionId, {'req', Req}, Context) ->
```

Comment:
Are there plans to use `Req` for something other than getting the IP? Meaning, can the change to `socketio_handler` be `{{IP, _}, Req1} = cowboy_req:peer(Req), Pid ! {'peer_ip', wh_util:to_binary(inet_parse:ntoa(IP))}` instead and this clause match the `peer_ip` tuple?